bindings/pyroot/pythonizations/python/ROOT/_pythonization/_roofit/_rooworkspace.py

```diff
@@ -11,6 +11,29 @@
 
 from ._utils import _kwargs_to_roocmdargs, cpp_signature
 
+def make_json_for_variable(var_name, var_dict):
+    val, max_value, min_value, is_constant = [None, None, None, True]
```

Comment:
```suggestion
    val = None
    max_value = None
    min_value = None
    is_constant = True
```
If you are not unpacking a tuple or list variable, it is preferred for readability to make only one assignment per line.

Can you change this, also in other places in this function?

Reply:
Done!